setuptools/msvc.py

```diff
@@ -1170,14 +1231,16 @@ def _unique_everseen(self, iterable, key=None):
                     seen_add(k)
                     yield element
 
-    def _get_content_dirname(self, path):
+    def _get_content_dirname(self, path, slash=True):
```

Comment:
Even better would be to unconditionally return the value without the slash and let the caller add a slash when appropriate.